framework/src/main/java/org/checkerframework/framework/type/DefaultTypeHierarchy.java

```diff
@@ -247,8 +249,13 @@ protected boolean isAnnoSubtype(
             @Nullable AnnotationMirror subtypeAnno,
             @Nullable AnnotationMirror supertypeAnno,
             boolean annosCanBeEmtpy) {
-        if (annosCanBeEmtpy && subtypeAnno == null && supertypeAnno == null) {
-            return true;
+        if (annosCanBeEmtpy) {
```

Comment:
Without this change, if only one of the annotations is null, ` qualifierHierarchy.isSubtype(subtypeAnno, supertypeAnno);` would throw an exception.  But the Javadoc says that `false` should be returned in that case, so I think this is a good change.

Reply:
If that is the case, then no client has ever taken advantage of this documented behavior.  Would it be better to simplify the specification rather than making the implementation more complex?  (I'm not sure which way I lean on that question.)

Reply:
I don't have a good argument for either option.  Really if both annotations are `null` this method shouldn't be called, so throwing an exception in that case makes sense.  So maybe it is better to change the documentation.

Reply:
Mention type variables in javadoc.  Change the Javadoc to match implementation.